src/refactorings/add-braces-to-if-statement/add-braces-to-if-statement.test.ts

```diff
@@ -0,0 +1,54 @@
+import { Editor, ErrorReason, Code } from "../../editor/editor";
+import { Selection } from "../../editor/selection";
+import { InMemoryEditor } from "../../editor/adapters/in-memory-editor";
+import { testEach } from "../../tests-helpers";
+
+import { addBracesToIfStatement } from "./add-braces-to-if-statement";
+
+describe("Add Braces To If Statement", () => {
+  let showErrorMessage: Editor["showError"];
+
+  beforeEach(() => {
+    showErrorMessage = jest.fn();
+  });
+
+  testEach<{ code: Code; selection?: Selection; expected: Code }>(
+    "should add braces to if statement",
+    [
+      {
+        description: "basic scenario",
+        code: "if (!isValid) return;",
+        selection: Selection.cursorAt(0, 17),
+        expected: `if (!isValid) {
+  return;
+}`
```

Comment:
👍 That's great!

Now can we think about other use-cases? What would happen for:

```js
if (isValid) 
  doSomething();
else
  doAnotherThing();
```

Ideally, it should only add braces to the `if` statement (= consequent), so I think it already works. But then, if we trigger that from the `else` (= alternate), I'd expect to add braces on the `else` statement instead.

That might be more tricky to implement. Let me know if you need help to figure this out. 

And this one should not change (which, I think, is already the case):

```js
if (isValid) {
  doSomething();
}
```

Reply:
Yes if have a full list of use-cases. My first goal was to get a first understanding of AST manipulation and get my first green test (TDD forever).